qa/TL0_python-self-test_xavier/test_nofw.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -e
 # used pip packages
-# due to https://github.com/numpy/numpy/issues/18131 we cannot use 1.19.5
-pip_packages='${python_test_runner_package} dataclasses numpy>=1.17,<=1.19.4 opencv-python pillow psutil'
+pip_packages='${python_test_runner_package} dataclasses numpy>=1.17 opencv-python pillow psutil'
```

Comment:
Regarding the comment and previous limitations, shouldn't it be `numpy>=1.20`?

Reply:
Good questions. For sure 1.19.5 didn't work, but I don't know if 1.20 does. All I know it works now whatever version is available there. So I would prefer not to add `>1.20` if we are not sure if that is the case.

Reply:
Up to you. My bet was for 1.20 because

https://github.com/numpy/numpy/issues/18131 is marked as fixed in https://github.com/numpy/numpy/pull/18196. The latter commit is merged in [1.20.0](https://github.com/numpy/numpy/releases/tag/v1.20.0) and there are no releases between 1.20.0 and 1.19.5.

Reply:
Done